odl/operator/default_ops.py

```diff
@@ -913,9 +812,9 @@ def __str__(self):
 
 class ResidualOperator(Operator):
 
-    """Operator that calculates the residual ``op(x) - vec``.
+    """Operator that calculates the residual ``op(x) - y``.
 
-    ``ResidualOperator(op, vector)(x) <==> op(x) - vec``
+    ``ResidualOperator(op, y)(x) <==> (x --> op(x) - y)``
```

Comment:
Addition: I see what you mean, you're right. I'll rather change the `x --> ...` ones so they match the "full evaluation" style.